INSTALL.md

```diff
@@ -40,7 +40,19 @@ The [libbpf](https://github.com/libbpf/libbpf) is a dependency of CNDP. Starting
 it can be installed using apt. For earlier Ubuntu versions, or for users who want the latest code,
 it can be installed from source.
 
-### Install libbpf from package manager
+#### _Note:_
+
+```bash
+Newer versions of libbpf greater than or equal to v0.7.0 require _libxdp_ to be installed. For now we
```

Comment:
This text is not a shell command so we don't need to enclose it within ```bash 

Italicizing libxdp didn't seem to work though: https://github.com/KeithWiles/cndp/blob/install_libbpf_update/INSTALL.md#note maybe because it's within the ``` bash block